tf-training/tf-job-operator/base/service.yaml

```diff
@@ -9,12 +10,9 @@ metadata:
       name: tfjobs-ui-mapping
       prefix: /tfjobs/
       rewrite: /tfjobs/
-      service: tf-job-dashboard.$(namespace)
-  name: tf-job-dashboard
+      service: tf-job-operator-dashboard.$(namespace)
```

Comment:
I think `tf-job-operator-dashboard` here should keep same as service.metadata.name, maybe we can use a var to make it work